src/components/button/_button.scss

```
@@ -30,6 +30,7 @@ button {
   min-width: 100px;
   overflow: hidden;
   position: relative;
+  text-overflow: ellipsis;
```

```suggestion
  text-overflow: ellipsis;
```

This all looks good now. But maybe we can add this under .contextualactionpanel because this effects all buttons there could be some add side effects?

---

And i think this is why these tests fail. https://travis-ci.com/github/infor-design/enterprise/jobs/300968163#L1373-L1378 so i would make this more specific to CAP Toolbar buttons by adding a rule there